refact_utils/finetune/utils.py

```diff
@@ -96,13 +96,18 @@ def add_result(k: str, model_dict: Dict):
         if model_dict.get('has_chat'):
             result['chat'].append(k)
 
+    def add_results_for_passthrough_provider(provider: str) -> None:
+        [
+            add_result(k, {'has_chat': 'chat' in v.get('filter_caps', [])})
+            for k, v in model_assigner.passthrough_mini_db.items()
+            if v.get('provider') == provider
+        ]
```

Comment:
* add_results_for_passthrough_provider -> _add_results_for_passthrough_provider
* write without list comprehension
* all passthrough models are about chat so no need in add_result usage here

Reply:
...and "chat" caps checking